better screenshot filename - show the filter type/filter id under the…

athomson0 opened the pull request:
… header

athomson0 commented:
Small change. There is now a sub-heading containing the filter type, along with the filter ID in the screenshot. Before:

![How much carbon is your digital advertising emitting (9)](https://user-images.githubusercontent.com/93927558/236775051-a868c2ee-a9d1-4378-a33a-82e19b19d3e2.png)


After:

![Campaign TlcM07wX - How much carbon is your digital advertising emitting (2)](https://user-images.githubusercontent.com/93927558/236775088-872bb0a4-0c0c-4feb-b1da-c3face07df0e.png)

These will also be present in the filename.

This should make it easier to tell which campaigns/tags screenshots belong to if a user is exporting lots of them.

laurenwilso commented:
@athomson0 love the idea of the campaign ID in the screenshot - can it also include the campaign name? Conscious that users often find the IDs themselves confusing (and if they're using the download in presentations etc, a full name as well as the ID would be even more helpful)

athomson0 commented:
> @athomson0 love the idea of the campaign ID in the screenshot - can it also include the campaign name? Conscious that users often find the IDs themselves confusing (and if they're using the download in presentations etc, a full name as well as the ID would be even more helpful)

Good point! I have updated it to show the item name instead of the ID. I can't imagine having the ID in the screenshot would be too useful for clients, so I have removed that (unless you think we should keep it in?)
